docs/settings/spaces-settings.asciidoc

```diff
@@ -0,0 +1,20 @@
+[role="xpack"]
+[[spaces-settings-kb]]
+=== Spaces Settings in Kibana
+++++
+<titleabbrev>Spaces Settings</titleabbrev>
+++++
+
+By default, Spaces is enabled in Kibana, and you can secure Spaces using 
+roles when Security is enabled.
+
+[float]
+[[spaces-settings]]
+==== Spaces Settings
+
+`xpack.spaces.enabled`::
+Set to `true` (default) to enable Spaces in {kib}.
+
+`xpack.spaces.maxSpaces`::
```

Comment:
If I understand the changeset correctly, this PR on its own doesn't prevent users from getting into a bad state, since `maxSpaces` isn't ever compared to the `index.max_result_window`. I think it'd be helpful to explain why this limit is in place, and what the benefits/risks are to adjusting this number.